### ISSUE TITLE
Double escaped quotes issue

### ISSUE THREAD

**Issue by migolo** [i18next/i18next-scanner#32]
#### Version
- i18next: <i>3.1.0</i>
- i18next-scanner: <i>1.4.0</i>

#### Configuration
```js
{ 
          lngs: ['en', 'es'], // supported languages
          fallbackLng: 'en',
          ns: ['mainsite'],
          defaultNs: 'mainsite',
          defaultValue: function(lng, ns, key) {
            if (lng === 'en') {
              // Return key as the default value for English language
              return key;
            }
            // Return the string '__NOT_TRANSLATED__' for other languages
            return '__NOT_TRANSLATED__';
          },
          nsSeparator: false,
          keySeparator: false,
          resource: {
              // the source path is relative to current working directory
              loadPath: 'translations/i18n/{{lng}}/{{ns}}.json',
              // the destination path is relative to your `gulp.dest()` path
              savePath: 'i18n/{{lng}}/{{ns}}.json'
          }
}
```
Having this on file:
```js
i18n.t("Primary \"email\" activation")
```

Produces on destination json:
```js
{ 
  "Primary \\\"email\\\" activation": "Primary \\\"email\\\" activation"
}
```

i18next searches for:
```js
{ 
  "Primary \"email\" activation": "Primary \"email\" activation"
}
```

**Comment by cheton** [comment]
Thanks for reporting the issue. It will be fixed soon in v1.4.1.

**Comment by cheton** [comment]
Published i18next-scanner@1.4.1 to npm

**Comment by migolo** [comment]
Confirmed solution, 
Thank you it works perfectly.
